parent/pom.xml

```diff
@@ -595,6 +595,14 @@
                 <scope>provided</scope>
             </dependency>
 
+            <!-- Content fragment API -->
+            <dependency>
+                <groupId>com.adobe.cq.dam</groupId>
+                <artifactId>cq-dam-cfm-api</artifactId>
+                <version>1.1.5-NPR-19008-R002</version>
```

Comment:
Is this available in 6.3 as well?

Reply:
NPR's `fixVersion` is set to 6.3.0.2, so I guess this can be answered with a yes (but to be sure I would have to double-check as I simply moved the dependency from the original extension pom).